7.application.md

```diff
@@ -31,116 +31,94 @@ This section defines the instances of components to create with the application.
-|-----------|------|----------|---------------|-------------|
-| `apiVersion` | `string` | Y | | The apiVersion of the Scope. |
-| `kind` | `string` | Y | | The kind of the Scope. |
-| `name` | `string` | Y | | The name of the Scope. |
-
-> The name field is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
 
 #### Trait
 
 The trait section defines an instance of a trait.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `name`| string | N | |  The name of trait definition. This is used to reference to the definition/schema of the Trait. For one type of trait, there could be only one config/deploy in one component. |
+| `type`| string | N | |  The name of trait definition. This is used to reference to the definition/schema of the Trait. For one type of trait, there could be only one config/deploy in one component. |
 | `properties`| [`Properties`](#properties) | N | |  The configurable properties that are exposed by this trait. |
 
-> The `name` field must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
+Notice that _Traits_ do not require a name since it is the OAM runtime the one responsible of instanciating the traits. The name of the component is expected to be used for the associated traits.
 
 
 ##### Properties
 
-A properties object for trait is an object whose structure is determined by the [definition reference](6.traits.md#DefinitionRef) of corresponding `TraitDefinition`. It may be a simple value, or it may be a complex object. Properties are validated against the schema appropriate for the trait.
+Properties specify the values that are associated with the entity attributes. When associated with _Components_ the values set or overwrite the component available parameters. Component parametrization is discussed on the [Component](3.component.md) entity spec. 
+
+Similarly, when properties are used on _Traits_ or _Scopes_ they set the values required by those entities to be instanciated. The structure is determined by the [definition reference](6.traits.md#DefinitionRef) of corresponding `TraitDefinition`. It may be a simple value, or it may be a complex object. Properties are validated against the schema appropriate for the trait or scope.
 
 ## Example
 
-The following is an example of a complete YAML file that expresses components and traits. This example illustrates the four configurational elements of a component: its name, workload settings, traits, and scopes.
+The following is an example of a complete YAML file that expresses components, traits, and scopes. This example illustrates the four configurational elements of a component: its type, properties, traits, and scopes.
```

Comment:
Should 'configurational' be 'configurable'?

Reply:
fixed